todo_app/tests/view_model_test.py

```diff
@@ -0,0 +1,106 @@
+import datetime
+from _pytest.fixtures import fixture
+from todo_app.view_model import ViewModel
+from todo_app.data.todo_item import TodoItem, TODO_STATUS, DOING_STATUS, DONE_STATUS
+import pytest
+import datetime
+from freezegun import freeze_time
+
+test_now = datetime.datetime(2021, 11, 11, 10, 00, 55)
+test_today = datetime.datetime(2021, 11, 11, 9, 00, 55)
+test_yesterday = datetime.datetime(2021, 11, 10, 10, 00, 55)
+
+@pytest.fixture
+def item_view_model():
+    items=[
+        TodoItem(0,"item to do", test_today ,TODO_STATUS),
+        TodoItem(1,"item to do #2",test_today,TODO_STATUS),
+        TodoItem(2,"item doing",test_today ,DOING_STATUS),
+        TodoItem(3,"item doing #2",test_today,DOING_STATUS),
+        TodoItem(4,"item done",test_today,DONE_STATUS),
+        TodoItem(5,"item done #2",test_today,DONE_STATUS),
+    ]
+    return ViewModel(items)
+
+
+def test_get_items(item_view_model):
+    assert len(item_view_model.items) == 6
+
+def test_get_todo(item_view_model):
+    items = item_view_model.todo_items
+
+    assert len(items) == 2
+    assert any(item.id == 0 for item in items)
+    assert any(item.id == 1 for item in items)
```

Comment:
One alternative to checking the ids would be to check the items that are returned have the expected status - it's slightly less brittle to people rearranging the items above